src/otx/core/data/dataset/base.py

```diff
@@ -30,15 +30,27 @@
 
 
 @dataclass
-class DataMetaInfo:
-    """Meta information of each subset datasets."""
+class LabelInfo:
+    """Object to represent label information."""
```

Comment:
What if LitModule requires other info such as data_root, subset names? I'm not sure whether there are such use cases, but if there are, should we create DatasetMetaInfo class which have LabelInfo attribute?

Reply:
We can rename and expand it if that case happens. Honestly, we already have the expanded one, `HLabelInfo`.